<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QDialog" name="Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StarCraft RMS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../RMS.ico</normaloff>../RMS.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="layoutFoldersToScan">
     <item>
      <widget class="QLabel" name="lblFoldersToScan">
       <property name="text">
        <string>Folders to Scan:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listFoldersToScan">
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutFolderAddRemove">
       <item>
        <widget class="QPushButton" name="btnAddFolder">
         <property name="text">
          <string>Add Folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemove">
         <property name="text">
          <string>Remove Folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutOutputFolder">
     <item>
      <widget class="QLabel" name="lblOutputFolder">
       <property name="text">
        <string>Output Folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leOutputFolder"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnOutputFolder">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbSelectNewMapsWhenStarCraftLaunches">
     <property name="text">
      <string>Select New Maps when StarCraft Launches</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutNumberOfRandomMaps">
     <item>
      <widget class="QLabel" name="lblNumberOfRandomMaps">
       <property name="text">
        <string>Number of Random Maps:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbNumberOfRandomMaps">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbGetNextMapInsteadOfRandom">
     <property name="text">
      <string>Get Next Map Instead of Random</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutLastMapPath">
     <item>
      <widget class="QLabel" name="lblLastMapPath">
       <property name="text">
        <string>Last Map Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leLastMapPath"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnLastMapPath">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbCheckPlayerCount">
     <property name="text">
      <string>Check Player Count (#)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutMinNumberOfPlayers">
     <item>
      <widget class="QLabel" name="lblMinNumberOfPlayers">
       <property name="text">
        <string>Min Number of Players:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbMinNumberOfPlayers">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutMaxNumberOfPlayers">
     <item>
      <widget class="QLabel" name="lblMaxNumberOfPlayers">
       <property name="text">
        <string>Max Number of Players:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbMaxNumberOfPlayers">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSaveAndRun">
       <property name="text">
        <string>Select New Maps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveAndClose">
       <property name="text">
        <string>Save and Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Discard and Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
